cros_gralloc/cros_gralloc_driver.cc

```diff
@@ -268,6 +268,17 @@ cros_gralloc_driver::cros_gralloc_driver()
 			DRV_INIT(drv_video_, gpu_grp_type, video_idx)
 		if ((virtio_node_idx != -1) && (virtio_node_idx != renderer_idx))
 			DRV_INIT(drv_kms_, gpu_grp_type, virtio_node_idx)
+		if (drv_kms_ && (virtio_node_idx != renderer_idx)) {
+			bool virtiogpu_with_blob = drv_virtgpu_with_blob(drv_kms_);
+			if (virtiogpu_with_blob) {
```

Comment:
Maybe add some comment here to clarify. Like: Only virtio device with blob func can share i915 bo to speed up, otherwise will use default virtio way for allocating scanout bo.

Reply:
they are in the commit message, do you mean we also add comment in code?

Reply:
Yeah. Just suggestions.